### PR TITLE
Release v0.27.0

### PR DESCRIPTION
**Pull request by knanao**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/content/en/blog/releases/v0.27.0.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+From this version it requires Piped with `v0.26.0` or over.
+
+### Notable Changes
+* Fix bug that the specified kubeConfigPath in Piped config was not used while handling manifests ([#3410](https://github.com/pipe-cd/pipecd/pull/3410))
+* Be able to show artifact versions with links ([#3382](https://github.com/pipe-cd/pipecd/pull/3382))
+* Show running version and documentation link on the web console ([#3328](https://github.com/pipe-cd/pipecd/pull/3328))
+* Improve false positives when using Secret with stringData in Kubernetes App ([#3299](https://github.com/pipe-cd/pipecd/pull/3299))
+* Fix a bug that previous plan-preview comments are not hidden if a customized bot name was used ([#3327](https://github.com/pipe-cd/pipecd/pull/3327))
+* Allow filtering suggested applications by using a new appSelector field in Piped config ([#3320](https://github.com/pipe-cd/pipecd/pull/3320))
+* Enable to view Application live state for Cloud Run ([#3301](https://github.com/pipe-cd/pipecd/pull/3301))
```
Comment (nghialv) [inline docs/content/en/blog/releases/v0.27.0.md:21]:
```suggestion
* Ability to view the live state of Cloud Run application ([#3301](https://github.com/pipe-cd/pipecd/pull/3301))
```

File: docs/content/en/blog/releases/v0.27.0.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+linkTitle: "Release v0.27.0"
+date: 2022-03-18
+description: >
+ Release v0.27.0
+---
+
+## Changelog since v0.26.0
+
+### Precondition
+From this version it requires Piped with `v0.26.0` or over.
```
Comment (nghialv) [inline docs/content/en/blog/releases/v0.27.0.md:12]:
```suggestion
From this version of the control plane, it requires Piped agent with a version greater than or equal to `v0.26.0`. 
```

Comment (knanao) [reply docs/content/en/blog/releases/v0.27.0.md]: Thanks! Addressed https://github.com/pipe-cd/pipecd/pull/3417/commits/7b55107ab628701b988f7dc536ac3f66258efb68

**Comment by nghialv** [comment]
Thank you.

/lgtm

File: docs/content/en/blog/releases/v0.27.0.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+## Changelog since v0.26.0
+
+### Precondition
+From this version of the control plane, it requires Piped agent with a version greater than or equal to `v0.26.0`. 
+
+### Notable Changes
+* Fix bug that the specified kubeConfigPath in Piped config was not used while handling manifests ([#3410](https://github.com/pipe-cd/pipecd/pull/3410))
+* Be able to show artifact versions with links ([#3382](https://github.com/pipe-cd/pipecd/pull/3382))
+* Show running version and documentation link on the web console ([#3328](https://github.com/pipe-cd/pipecd/pull/3328))
```
Comment (khanhtc1202) [inline docs/content/en/blog/releases/v0.27.0.md:17]:
```suggestion
* Show control-plane running version and documentation link on the web console ([#3328](https://github.com/pipe-cd/pipecd/pull/3328))
```
nits

**Comment by khanhtc1202** [comment]
Thank you 🙌 
/approve

**Pull request merged**